_2020_pages/agenda.html

```diff
@@ -0,0 +1,161 @@
+
+        </p>
+<!--
+        <h2>Day 1 - Monday, 28 September 2020</h2>
+        <table class="agenda">
+            <thead>
+                <tr>
+                    <th class="row-1 agenda-time"><center>Time</center></th>
+                    <th class="row-2 agenda-activity"><center>Activity</center></th>
+                </tr>
+            </thead>
+
+            {% for agenda in site.data.twenty_20.agendas_1 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table>
+
+        <h2>Day 2 - Tuesday, 29 September 2020</h2>
+        <table class="agenda">
+            <thead>
+                <tr>
+                    <th class="row-1 agenda-time"><center>Time</center></th>
+                    <th class="row-2 agenda-activity"><center>Activity</center></th>
+                </tr>
+
+            </thead>
+
+            {% for agenda in site.data.twenty_20.agendas_2 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table>
+
+        <h2>Day 3 - Wednesday, 30 September 2020</h2>
+        <table class="agenda">
+            <thead>
+                <tr>
+                    <th class="row-1 agenda-time"><center>Time</center></th>
+                    <th class="row-2 agenda-activity"><center>Activity</center></th>
+                </tr>
+
+            </thead>
+
+            {% for agenda in site.data.twenty_20.agendas_3 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table>
+
+        <h2>Day 4 - Thursday, 01 October 2020</h2>
+        <table class="agenda">
+            <thead>
+                <tr>
+                    <th class="row-1 agenda-time"><center>Time</center></th>
+                    <th class="row-2 agenda-activity"><center>Activity</center></th>
+                </tr>
+
+            </thead>
+
+            {% for agenda in site.data.twenty_20.agendas_4 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table>
+
+        <h2>Day 5 - Friday, 02 October 2020</h2>
+        <table class="agenda">
+            <thead>
+                <tr>
+                    <th class="row-1 agenda-time"><center>Time</center></th>
+                    <th class="row-2 agenda-activity"><center>Activity</center></th>
+                </tr>
+
+            </thead>
+
+            {% for agenda in site.data.twenty_20.agendas_5 %}
+            <tr>
+                <td>{{ agenda.time }}</td>
+                <td>{{ agenda.info }}</td>
+            </tr>
+            {% endfor %}
+        </table> -->
+
+        <h2>School materials</h2>
+        <ul>
```

Comment:
Maybe also comment out this part? Or add a remark that school materials will be added here?

Reply:
Right.